src/test/testGraphs.ts

```diff
@@ -46,60 +62,31 @@ const adr6 = getPrefabDsnpUserId(6);
  * Prefabs are meant to work with other prefab components
  */
 
-export const getPreFabSocialGraph = (): Graph[] => {
-  const socialGraph = [
-    {
-      dsnpUserId: adr0,
-      following: [adr1, adr2],
-      followers: [adr1, adr6],
-    },
-    {
-      dsnpUserId: adr1,
-      following: [adr0, adr6],
-      followers: [adr0, adr2],
-    },
-    {
-      dsnpUserId: adr2,
-      following: [adr0, adr1, adr3, adr4, adr5, adr6],
-      followers: [],
+export const getPreFabSocialGraph = (): SocialGraph => {
+  const following = {
+    [adr0]: { [adr1]: true, [adr2]: true },
```

Comment:
Can we have one of these, plus a test, with 3 followers? There can be bugs where 1 item or 2 items works but it breaks on  3 items.  3 is more N-like than 1 or 2.

Reply:
Sure. I'm not sure I agree 3 is significantly more n-like though.